playbooks/templates/.env.j2

```diff
@@ -1,118 +1,237 @@
+# Certificate string that will be used to generate ssl certificates, read more in docker/caddy/Caddyfile
+# xxxqqq: Deprecated
+{% if portal_cluster_id is defined %}
+SSL_CERTIFICATE_STRING={{ cluster_domain }}, *.{{ cluster_domain }}, *.hns.{{ cluster_domain }}, *.{{ server_domain }}, *.hns.{{ server_domain }}
+{% else %}
+SSL_CERTIFICATE_STRING={{ server_domain }}, *.{{ server_domain }}, *.hns.{{ server_domain }}
+{% endif %}
 
-# (optional) the port on which siad is listening, defaults to 9980
+# (optional) The port on which siad is listening, defaults to 9980.
 API_PORT={{ webportal_server_config.sia_api_port }}
 
-# this is auto generated secure key for your handshake service integration
+# Auto generated secure key for your Handshake service integration.
 HSD_API_KEY={{ webportal_server_config.hsd_api_key }}
 
 ###############################################################################
 # Common
 
-# absolute url to the portal api ie. https://siasky.net (general portal
-# address)
-# xxx update for cluster
-SKYNET_PORTAL_API=https://{{ webportal_server_config.domain_name }}
-
-# this is the administrator contact email you need to supply for communication
-# regarding SSL certification
+# Absolute url to the (cluster) portal API.
+# For a single server: https://single-portal.com
+# For a cluster: https://cluster.com
+SKYNET_PORTAL_API=https://{{
+    cluster_domain
+    if portal_cluster_id is defined
+    else server_domain
+}}
+
+# Administrator contact email.
+# Needed for communication regarding SSL certification.
+# xxxqqq: Deprecated.
 EMAIL_ADDRESS={{ skynetlabs_devs_email }}
 
-# (optional) if using cloudflare as dns loadbalancer (need to change it in
-# Caddyfile too)
+# (optional) If using CloudFlare as DNS loadbalancer (need to change it in
+# Caddyfile too).
+# xxx: Not implemented in Ansible
 CLOUDFLARE_AUTH_TOKEN=
 
-# (optional) if using route53 as a dns loadbalancer
+# (optional) If using Route53 as a DNS loadbalancer.
 AWS_ACCESS_KEY_ID={{ webportal_common_config.aws_access_key}}
 
-# (optional) if using route53 as a dns loadbalancer
+# (optional) If using Route53 as a DNS loadbalancer.
 AWS_SECRET_ACCESS_KEY={{ webportal_common_config.aws_secret_access_key }}
 
-# (optional) only required if you're using the discord notifications
-# integration
+# (optional) Only required if you're using the Discord notifications
+# integration.
+# xxx move those to common/cluster configs in LastPass
 DISCORD_WEBHOOK_URL={{ discord_webhook_url | default('') }}
 DISCORD_MENTION_ROLE_ID={{ discord_mention_role_id | default('') }}
 DISCORD_MENTION_USER_ID={{ discord_mention_user_id | default('') }}
 
 # common path for cluster, single path for each dev server
-S3_BACKUP_PATH={{ webportal_common_config.s3_backup_path if ('webportals_prod' in group_names) else (webportal_common_config.s3_backup_path + '_' + inventory_hostname) }}
+# S3 backup path.
```

Comment:
@ro-tex the S3 backup is for mongo right? Anything else?

We should add to the comment what S3 is being used for.

Reply:
Then I would also list it as optional and we should link to what is triggering the backups. 

Fine for a follow up, just thinking through questions a community portal operator might have. They might want to back it up somewhere else and they should be able to.

Reply:
It's for Mongo and Cockroach right now but it will be just Mongo once we get rid of Kratos.

I agree that it should be marked as optional and that we should link to an explanation about backups.

Reply:
Created issue #104